Makefile

```diff
@@ -52,6 +52,22 @@ libs: ./include/rule_type.h src/parser.c src/dialect.c $(SRC_FILES) src/Makefile
 	cd src; $(MAKE) CC=$(CC) $@
 .PHONY: libs
 
+libs_so: ./include/rule_type.h src/parser.c src/dialect.c $(SRC_FILES) src/Makefile
+	cd src; $(MAKE) CC=$(CC) $@
+.PHONY: libs_so
+
+install: libs_so
+	cp ./libs/libgherkin.so.1.0 /usr/lib/libgherkin.so.1.0
```

Comment:
.1.0? Shouldn't it rather be the gherkin version?

Reply:
I suggest we add a `VERSION` file with a single line containing the semver version. The upcoming version is `5.0.1`. Then the `Makefile` can just `cat` that file.